libs/npm/core/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "tno-core",
   "description": "TNO shared library",
-  "version": "0.0.274",
+  "version": "0.0.276",
```

Comment:
Any reasons why we use 276 instead of 275 here?

Reply:
I somehow managed to push a version of the package which didn't have all the changes in it.  I suspect it was not having saved a file in the IDE and then doing the push.  I tried to unpublish that version but NPM wouldn't let me so I had to push another patch version again.